src/js/components/FormField/__tests__/__snapshots__/FormField-test.js.snap

```diff
@@ -522,7 +434,6 @@ exports[`FormField contentProps 1`] = `
           <input
             autocomplete="off"
             class="c5"
-            value=""
```

Comment:
@jcfilben any concerns around this attribute being removed? I'm going to look into it a bit more

Reply:
I looked a little more, this is a change in the DOM although there's no break to functionality. This is in cases where the input is in a form and the input value is never updating in the DOM since it's managed by the form itself (this is the case regardless of styled-components version) so it moreso seems like it's just cleaning up an empty DOM attribute?

Reply:
Agree that it looks like a cleanup. There are other snapshots in this PR that have the same change. It looks like styled components is just stricter and doesn't let an empty value pass into the DOM. I think this snapshot change is fine

Reply:
I feel comfortable with this cleanup. It mirrors the behavior of [Radix forms](https://www.radix-ui.com/primitives/docs/components/form) as well.